src/org/zaproxy/zap/extension/zest/ZapAddOn.xml

```diff
@@ -1,12 +1,13 @@
 <zapaddon>
 	<name>Zest - Graphical Security Scripting Language</name>
-	<version>28</version>
+	<version>29</version>
```

Comment:
Looks like 28 hadn't been released yet. So this could stay.
https://github.com/zaproxy/zap-extensions/releases